app/models/mixins/relationship_mixin.rb

```diff
@@ -664,10 +729,14 @@ def remove_all_relationships(*rels)
   end
 
   def is_descendant_of?(obj)
+    return ancestor_ids.include?(obj.id) if use_ancestry?
```

Comment:
nice

is there a reason not to use ancestry's build in methods that you so nicely corrected in the documentation?

|method             |return value|
|-------------------|---------------|
|`parent_of?(node)`  | node's parent is this record|
|`root_of?(node)`    | node's root is this record|
|`ancestor_of?(node)`| node's ancestors include this record|
|`child_of?(node)`   | node is record's parent|
|`descendant_of?(node)` | node is one of this record's ancestors|
|`indirect_of?(node)` | node is one of this record's ancestors but not a parent|

Reply:
Moving this answer so I can find it later: the mixin `is_ancestor_of?` calls the gem `ancestor_of?` which is `node.ancestor_ids.include?(self.id)` but that `ancestor_ids` is calling the mixin method `ancestor_ids`, not the gem method, and will return nothing because by that point our `relationship_type` is not set.